build.gradle

```diff
@@ -46,6 +46,11 @@ task clean (type: GradleBuild) {
   tasks = ['clean']
 }
 
+task realclean (type: GradleBuild) {
```

Comment:
Hmm... I wonder if there's a better name for this?

Or... is there a reason one might want to `clean` without doing a `realclean`?  If not, maybe we could just add it to the normal clean?